_config.yml

```diff
@@ -34,6 +34,17 @@ report_diff_filename: _cache/_deploy_index_report_diff.yaml
 debbuild_release_path: _cache/debbuild_releases
 wiki_title_index_filename: _cache/ros_org_wiki_title_index.txt
 
+# If true, this skips finding repos based on the repo sources
+skip_discover: false
+# If true, this skips updating the known repos
+skip_update: false
+# If true, this skips scraping the cloned repos
+skip_scrape: false
+# If true, this skips generating the search index
+skip_search_index: false
```

Comment:
I believe the `_skip*` variables had been removed because they will always be overridden, since this file is never used alone. Is the intention to set these as default values? Maybe these could be in the plugin (if that's not already the current behaviour)?

I imagine someone may be confused changing the value here and not seeing any change in results.

Reply:
It is now used on its own for `make build`. The problem with `build` being a rule that depends on the others is that skipping is really just skipping rosindex_generator plugin sections, but it cannot skip the Jekyll build itself. So right now, before this PR, `make build` would write everything to `site` about 4 times. This solves that.

Reply:
Oh you're right. Thanks!